server/monitor.ts

```diff
@@ -53,6 +53,7 @@ if (('Config' in global) &&
 
 export const Monitor = new class {
 	connections = new TimedCounter();
+	commands = new TimedCounter();
```

Comment:
I think `netRequests` and `countNetRequest` makes more sense? We're not counting _all_ commands, after all.